content/learning-paths/smartphones-and-mobile/debugging_with_mte_on_pixel8/_review.md

```diff
@@ -0,0 +1,60 @@
+---
+review:
+    - questions:
+        question: >
+             Which of the below statements is true?
+        answers:
+            - Memory safety bugs are errors in handling memory by the hardware. Although important, they are not very common in the Android codebase.
+            - Memory safety bugs are errors in handling memory by the software. Although important, they are not very common in the Android codebase.
+            - Memory safety bugs are errors in handling memory by the software. They are important and very common in the Android codebase.
+            
+        correct_answer: 3                    
+        explanation: >
+            Memory safety bugs are errors in handling memory by software. Memory safety bugs are the most common issue in the Android codebases. They account for over 70% of high severity security vulnerabilities and for millions of user-visible crashes.
+
+    - questions:
+        question: >
+           Which of the below statements is false?
+        answers:
+            - The MTE Test application implements common memory safety bugs.
```

Comment:
not sure if you need this question, but if so, you only need one of the first 2 wrong answers - 2 similar!